addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -1959,6 +1960,23 @@ export class OdooEditor extends EventTarget {
         ) {
             const restore = preserveCursor(this.document);
             this.observerFlush();
+            // if the delete range create and empty block,
+            // and the range end in another element,
+            // we delete the block and move the cursor to the next block.
```

Comment:
This is kind of weird... It means the code is like this:
```js
doSomething();
if (weDidTheWrongThing)  {
    ... // Fix what we did.
}
```
I think the fix should be in `doSomething` so that we never do the wrong thing in the first place, rather than fixing it later on as an afterthought.

Reply:
where else would you expect this code ?
I'm in the loop that do the deleteRange ...
and the deleteRange is just a loop of `oDeleteBackward` which is agnostic of deleteRange context.

Reply:
> I'm sure your premium IDE supports auto-wrapping your comments to 80 chars on a shortcut... 
yes it does, but the "only on changes line" options was broken last time I checked and then reformat the all file.
So I have to do it manually and I forget, because it doesn't bother me.